libcrux-ml-kem/src/vector/portable/arithmetic.rs

```diff
@@ -33,7 +35,8 @@ pub(crate) fn get_n_least_significant_bits(n: u8, value: u32) -> u32 {
 #[inline(always)]
 pub fn add(mut lhs: PortableVector, rhs: &PortableVector) -> PortableVector {
     for i in 0..FIELD_ELEMENTS_IN_VECTOR {
-        lhs.elements[i] += rhs.elements[i];
+        hax_lib::debug_assert!(lhs.elements[i] <= i16::MAX - rhs.elements[i] && lhs.elements[i] >= i16::MIN - rhs.elements[i]);
+        lhs.elements[i] = lhs.elements[i].wrapping_add(rhs.elements[i]);
```

Comment:
I don't think that we want a `wrapping_add` here. We need to show that this is not panicking, hence not wrapping. Same goes for the other operations.